### DISCUSSION TITLE
Training (A+B) steps one time is equal to A steps trained and another B steps under same learning rate setting?

### DISCUSSION THREAD

**Discussion by hrs97** [TheLastBen/fast-stable-diffusion#1883] [Q&A]
Using dreambooth for training,I have a dataset of 500 pics, which includes 5 concepts. Firstly, I trained it with 25000 steps, and it's performance was not so good, so i resumed training another 25000 steps, the model failed and outputed a mess. And then, i trained the model with 500 pics 50000 steps one time, it performed well. But i have no idea if it will get better after another steps. 
So i have two questiones, the first is training a model with (A+B) steps one time performs as well as A steps trained and another B steps? The second is if resuming training for another too many steps such as 10000 steps will destroy the model's performance?
Due to the consumption of computational unit and runtime limits in colab, i can't train a model from zero with too too too many steps everytime to see if it get better.

**Comment by TheLastBen** [comment]
50000 steps is enough to train a new model from scratch.
for each style/concept, pick 10 images, train with the default settings, that's it, it's better to use a model for each style

**Reply by hrs97** [reply]
Thanks!
